adr/0004-webscraping.md

```diff
@@ -25,6 +29,10 @@ Webscraping comes with the following downsides:
 - It will still be possible to have custom integrations provide information via webscraping
 - Generic integration to parse HTML are excluded from this decision
 
+### Exception
```

Comment:
We already have an exception "Generic integration to parse HTML are excluded from this decision"
Should not the new "Exception" section should include both?